docs/deploy-using-sepolia.org

```diff
@@ -169,7 +168,7 @@ progressing as expected.
 
 #+begin_src bash
 rpc_url=$(kurtosis port print cdk cdk-erigon-node-001 rpc)
-zkevm_l2_admin_private_key="0x69659a88bd0950e4914f69aaa867e258ff39bc6274d1576a23c477c65987e31f"
+zkevm_l2_admin_private_key="0x69659a88bd0950e4914f69aaa867e258ff39bc6274d1576a23c477c65987e31f" # You might have to change this value!
```

Comment:
you could replace this with something like `$(yq '.args.zkevm_l2_admin_private_key' .github/tests/external-l1/deploy-cdk-to-sepolia.yml)` now i guess?

Reply:
Addressed by [docs: retrieve zkevm_l2_admin_private_key from test file](https://github.com/0xPolygon/kurtosis-cdk/pull/320/commits/1c244b428ba2e12f89b3d7c828b4de309aeb484a)